core/src/main/java/com/github/szsalyi/flashcardapp/entity/User.java

```diff
@@ -0,0 +1,64 @@
+package com.github.szsalyi.flashcardapp.entity;
+
+import com.github.szsalyi.flashcardapp.entity.enums.Role;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Created by ssalyi on 8/14/2017.
+ */
+public class User implements Serializable {
```

Comment:
Missing `@Entity`.
Rename the class to something like `UserEntity` to distinguish it from domain objects of upper layers.
Use the `@Table` annotation to specify the name of the database table for this entity (like 'users').

Use Lombok to generate boilerplate code.